activiti-core/activiti-engine/src/main/java/org/activiti/engine/delegate/DelegateExecution.java

```diff
@@ -182,4 +184,8 @@ public interface DelegateExecution extends VariableScope {
    */
   void setMultiInstanceRoot(boolean isMultiInstanceRoot);
 
+  default ProcessEngineConfigurationImpl getEngineServices() {
```

Comment:
can we return `ProcessEngineConfiguration` instead and have the implementation classes implement it? We should not expose `impl` classes here.

Reply:
You're right, I should be using `ProcessEngineConfiguration` instead.. noob mistake. Fixed.